README.md

```diff
@@ -139,6 +139,15 @@ If you plan to release your application you'll need to publish it using the same
 
 ## Example applications
 
+As Decidim is a ruby gem, you can see the [https://github.com/decidim/decidim/network/dependents?type=application](dependent repostiories) to see how many applications are on the wild or tests that others developers have made. Here's a partial list with some of the projects that have used Decidim: 
+
 * [Demo](http://staging.decidim.codegram.com)
-* [Decidim Barcelona](https://decidim.barcelona)
-* [L'H ON dels barris](https://www.lhon-participa.cat)
+* [Decidim Barcelona](https://decidim.barcelona) - [https://github.com/AjuntamentdeBarcelona/decidim-barcelona](View Github)
```

Comment:
Format should be `[text of the link](URL)` :) So `[View on GitHub](https://github.com/AjuntamentdeBarcelona/decidim-barcelona)` should be the correct way :)